crates/ruff_linter/src/rules/pylint/rules/no_method_decorator.rs

```diff
@@ -0,0 +1,170 @@
+use std::collections::HashMap;
+
+use ruff_diagnostics::{AlwaysFixableViolation, Diagnostic, DiagnosticKind, Edit, Fix};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::{self as ast, Expr, Stmt};
+use ruff_python_trivia::indentation_at_offset;
+use ruff_text_size::{Ranged, TextRange, TextSize};
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks for the use of a classmethod being made without the decorator.
```

Comment:
It would be nice to add some examples here. Personally I don't even know how you define a classmethod without a decorator.  😊

(just a random observer here who's interested in new Pylint rules, thanks for working on this!)

Reply:
Ah, my mistake!